docs/Execution.md

```diff
@@ -1,92 +1,133 @@
+- Send a PubSub message to FCA's client follwoing the instructions in the [PubSub Handlers documentation](pubSubHandlers/PubSubHandlers.md).
+
+- **Sample Intent Message**:
+A sample intent message to trigger FCA remotely is provided below:
+
+```
+{
+    "jsonrpc": "2.0",
+    "id": 1,
+    "method": "APIName",
+    "params": {
+        "appId": "AppId",
+        "intent": {
+            "action": "search",
+            "data": {
+                "query": {
+                    "task": "runTest",
+                    "params": {
+                        "certification": true,
+                        "exceptionMethods": [
+                            {
+                                "method": "Api.method",
+                                "param": {
+                                    "type": "distributor"
+                                }
+                            },
+                            "..."
+                        ],
+                        "methodsToBeExcluded": [
+                            "Api.method",
+                            "Api.method",
+                            "..."
+                        ],
+                        "modulesToBeExcluded": []
+                    },
+                    "action": "CORE",
+                    "context": {
+                        "communicationMode": "SDK"
+                    },
+                    "metadata": {
+                        "target": "targetName",
+                        "targetVersion": "targetVersion",
+                        "...": "..."
+                    },
+                    "asynchronous": false,
+                    "appType": "firebolt"
+                }
+            },
+            "context": {
+                "source": "device"
+            }
+        }
+    }
+}
+
+```
+
+for more specific triggering follow instructions 
+[RunTest Handler documentation](intentReaderHandlers/RunTestHandler.md).
+
+
+# Test Types
+
+## Sanity
+
+Sanity testing ensures that all APIs function correctly at a basic level, serving as a quick check method.
+Execute the Sanity Test Suite from the UI, as described in the 'Supported Ways of Execution' section.
+This suite tests all APIs by invoking predefined calls under each API, offering a comprehensive overview of basic functionality.
+
+## Regression
+
+Regression testing involves in-depth testing to ensure new features or changes haven't adversely affected existing functionality.
+Create custom test cases with specific parameters tailored to the features or changes being tested.
+Send requests with custom parameters and verify responses against predefined data sets for targeted testing.
+to trigger those tests, build a framework communicating with your device, and send requests and verify responses as needed.
+
+
+# Modes of communication
```

Comment:
Capitalize Communication

Reply:
@ksentak for this one Modes of Communication is introduced as h1 in the TOC. should we change that ?

Reply:
Seconded, it should be 
```
# Title
## TOC
## Section
```

Since we have sub sections aside from `Background`/`Setup`/`Usage` I'm thinking maybe we just make them bold because h4 looks off. So like:
```
# FCA: Executing Tests
## Table of Contents
## Supported ways of Execution
### UI
**Background**:
**Setup**:
**Usage**:
etc...
```

Reply:
updated to 
## Modes of communication
also updated all the other titles in the file according to 
# FCA: Executing Tests
## Table of Contents
## Supported ways of Execution
### UI
**Background**:
**Setup**:
**Usage**:
etc...